content/departments/product-engineering/engineering/cloud/devops/managed/delete_process.md

```diff
@@ -250,17 +255,6 @@ terraform apply
 cd ../ # back to the repo root directory
 ```
 
-### Remove the GCP project
-
-```
-cd infrastructure/gcp/projects
-cat .tool-versions
-git checkout main
-git pull
-terraform apply
-cd ../ # back to the repo root directory
-```
```

Comment:
This already happens earlier in the process (needs to be applied straight away because the statefile needs to be checked in before merging)